warehouse/helpdesk/interfaces.py

```diff
@@ -0,0 +1,16 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from zope.interface import Interface
+
+
+class IHelpDeskService(Interface): ...
```

Comment:
Note to self: probably need to actually write the interface functions here, but nothing in linter-land seems to care? 🤔

Reply:
Yep, that was it. I've added in 28462ff9de631a94f3a99c1bdd1eef64a87f34d2